.github/workflows/build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
 
     - name: Test
       run: make coverage
-      if: matrix.os == 'ubuntu-latest'
+      if: matrix.os != 'windows-latest'
```

Comment:
runs local filesystem tests on mac, I validated docker tests (samba and s3) locally but unfortunately no more docker support on mac arm on GHA due to licensing issues.